lib/logidze/ignore_log_data.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Logidze
+  # Add `has_logidze` method to AR::Base
+  module IgnoreLogData
+    extend ActiveSupport::Concern
+
+    module BackportIgnoredColumns # :nodoc:
+      attr_writer :ignored_columns
+
+      def all(*args)
+        super(*args).select(attribute_names - ignored_columns)
+      end
+
+      def ignored_columns
+        @ignored_columns ||= []
+      end
+    end
+
+    def self.included(base)
+      base.extend BackportIgnoredColumns if Rails::VERSION::MAJOR < 5
+      base.ignored_columns += ["log_data"]
+    end
+
+    module ClassMethods # :nodoc:
+      def with_log_data
+        select(attribute_names + ["log_data"])
+      end
+    end
+
+    def log_data
+      return log_data_attribute if log_data_attribute
+
+      raise ActiveModel::MissingAttributeError
+    end
+
+    def log_data!
+      log_data_attribute || self.class.with_log_data.find(id).log_data
```

Comment:
We shouldn’t call DB twice if we call `log_data!` twice.

Also, can we use `pluck(:log_data)` to avoid additional AR instance initialization?